File: CompSci 1/Unit 12, Selection/Triangle/Triangle.java
import java.util.Scanner;
public class Triangle {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter Side 1 --> ");
        int side1 = sc.nextInt();

        System.out.print("Enter Side 2 --> ");
        int side2 = sc.nextInt();

        System.out.print("Enter Side 3 --> ");
        int side3 = sc.nextInt();

        System.out.print("\nThis triangle is a");

        if (side1 == side2 && side1 == side3) {
            System.out.println("n equilateral");
        }
        if (side1 != side2 && side2 != side3 && side1 != side3) {
            System.out.println(" scalene");
        }
        else {
            System.out.println("n isosceles");
        }
    }

}

File: CompSci 1/Unit 12, Selection/Triangle/Triangle.kt
import java.util.Scanner

fun main(args: Array<String>) {
    val sc = Scanner(System.`in`)

    print("Enter Side 1 --> ")
    val side1 = sc.nextInt()

    print("Enter Side 2 --> ")
    val side2 = sc.nextInt()

    print("Enter Side 3 --> ")
    val side3 = sc.nextInt()

    print("\nThis triangle is a")

    if (side1 == side2 && side1 == side3) {
        println("n equilateral")
    }
    if (side1 != side2 && side2 != side3 && side1 != side3) {
        println(" scalene")
    } else {
        println("n isosceles")
    }
}